terraform/modules/daily_snapshot/service/daily_snapshot.rb

```diff
@@ -66,3 +66,6 @@ def latest_snapshot_date(chain_name = 'calibnet')
 [LOG_EXPORT_SCRIPT_RUN, LOG_EXPORT_DAEMON, LOG_EXPORT_METRICS].each do |log_file|
   puts "Snapshot export log:\n#{File.read(log_file)}\n\n" if File.exist?(log_file)
 end
+
+# Clean up daemon logs after everything is done
+File.delete(LOG_EXPORT_DAEMON) if File.exist?(LOG_EXPORT_DAEMON)
```

Comment:
Why?

Reply:
No. I mean't the logs themselves, as in removal of lines that are useless/redundant/noise. The file should stay.